week-2/F-logical-operators/exercise2.js

```diff
@@ -5,7 +5,22 @@
   Update the code so that you get the expected result.
 */
 
-function isNegative() {}
+function isNegative(num) {
+  return num < 0;
+}
+
+function isBetween5and10(num) {
+  return num >= 5 && num <= 10;
+}
+startsWithD;
```

Comment:
What's `startsWithD`? Looks like something left over from a previous version. Always check over your code and make sure it runs before committing it :slightly_smiling_face: .